### DISCUSSION TITLE
Does the PEFT package have nightly versions that are pushed and can be installed?

### DISCUSSION THREAD

**Discussion by arnavgarg1** [huggingface/peft#1018] [General]
Same as the title: Does the PEFT package have nightly versions that are pushed and can be installed?

Some background:
I work on another open-source package that uses PEFT for fine-tuning. I want to use LoRA with Mistral-7b but it seems like this currently only works with PEFT's master branch, but not with the latest stable release. I would love to pin the open-source package  I work on to a stable nightly instead of PEFT master until the next stable release.

**Comment by BenjaminBossan** [comment]
No, we don't have a nightly release. For my understanding, what would be the advantage to you to install from a nightly package instead of installing from main?

**Comment by arnavgarg1** [comment]
@BenjaminBossan I find that master branches tend to have changes that are often breaking changes, and I wanted to pin Ludwig to a specific nightly release that I know is stable while I wait for peft 0.6 to be released.

**Reply by BenjaminBossan** [reply]
I see. You could install from a specific commit hash too instead of pulling the latest main branch.

**Reply by arnavgarg1** [reply]
Makes sense - I didn't consider that, will do that instead! 

Do we know when PEFT 0.6 might come out?

**Reply by BenjaminBossan** [reply]
Can't make any promises, but our goal is within 2 weeks.

**Reply by arnavgarg1** [reply]
Sounds great, thanks for all the hard work @BenjaminBossan!
